drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -15,7 +15,7 @@ class Forms::PickListOption < Types::BaseObject
     field :group_label, String, 'Label for group that option belongs to, if grouped', null: true
     field :initial_selected, Boolean, 'Whether option is selected by default', null: true
 
-    def self.options_for_type(pick_list_type, user:, project_id: nil)
+    def self.options_for_type(pick_list_type, user:, project_id: nil, inventory_id: nil)
```

Comment:
would it make sense to have `inventory_id` be something more generic, like `parent_id` or `source_id`?  Then dependent on the pick-list type you could use the same variable to hold the relation.  That'd work for `project_id` as well I think.